### PR TITLE
Add documentation for AVM fritzbox smarthome component

### PR DESCRIPTION
**Pull request by hthiery**
**Description:** Documentation for the fritzbox component and platform

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#10688

## Checklist:

  - [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/climate.fritzhome.markdown [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: "Fritzhome Thermostat"
```
Comment (c727) [inline source/_components/climate.fritzhome.markdown]: Fritzbox

File: source/_components/climate.fritzhome.markdown [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+description: "Instructions how to integrate the AVM fritzhome thermostat."
+date: 2017-11-12 17:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: avm.png
+ha_category: Hub
+ha_release: 0.59
```
Comment (c727) [inline source/_components/climate.fritzhome.markdown]: 0.64

File: source/_components/climate.fritzhome.markdown [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+ha_iot_class: "Local Polling"
+---
+
+<p class='note'>
+To get AVM fritzhome thermostat follow the instructions for the general [Fritzhome](/components/fritzhome/).
```
Comment (c727) [inline source/_components/climate.fritzhome.markdown]: Fritzbox

File: source/_components/fritzhome.markdown [hunk 4]
```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: "Fritzbox"
+description: "Instructions how to integrate the AVM fritzbox smarthome components."
```
Comment (c727) [inline source/_components/fritzhome.markdown]: Fritzbox

File: source/_components/fritzhome.markdown [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+fritzbox:
+  devices:
+    - host: fritz.box
+      username: YOUR_USERNAME
+      passowrd: YOUR_PASSWORD
+```
+
+{% configuration %}
+  devices:
+    description: A list of fritzbox devices.
```
Comment (c727) [inline source/_components/fritzhome.markdown]: Fritzbox

File: source/_components/fritzhome.markdown [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    required: true
+    type: map
+    keys:
+      host:
+        description: The hostname or IP address of the fritzbox.
+        required: true
+        type: string
```
Comment (c727) [inline source/_components/fritzhome.markdown]: Optional

File: source/_components/fritzhome.markdown [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+```
+
+{% configuration %}
+  devices:
+    description: A list of fritzbox devices.
+    required: true
+    type: map
+    keys:
+      host:
+        description: The hostname or IP address of the fritzbox.
```
Comment (c727) [inline source/_components/fritzhome.markdown]: Fritzbox

File: source/_components/switch.fritzhome.markdown [hunk 8]
```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: "Fritzhome Switch"
```
Comment (c727) [inline source/_components/switch.fritzhome.markdown]: Fritzbox

File: source/_components/switch.fritzhome.markdown [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+description: "Instructions how to integrate the AVM fritzhome switch."
```
Comment (c727) [inline source/_components/switch.fritzhome.markdown]: Fritzbox

File: source/_components/switch.fritzhome.markdown [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+date: 2017-11-12 17:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: avm.png
+ha_category: Hub
+ha_release: 0.59
```
Comment (c727) [inline source/_components/switch.fritzhome.markdown]: 0.64

File: source/_components/switch.fritzhome.markdown [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+layout: page
+title: "Fritzhome Switch"
+description: "Instructions how to integrate the AVM fritzhome switch."
+date: 2017-11-12 17:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: avm.png
+ha_category: Hub
```
Comment (c727) [inline source/_components/switch.fritzhome.markdown]: Switch

File: source/_components/climate.fritzhome.markdown [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+layout: page
+title: "Fritzhome Thermostat"
+description: "Instructions how to integrate the AVM fritzhome thermostat."
+date: 2017-11-12 17:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: avm.png
+ha_category: Hub
```
Comment (c727) [inline source/_components/climate.fritzhome.markdown]: Climate

File: source/_components/switch.fritzhome.markdown [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+ha_release: 0.59
+ha_iot_class: "Local Polling"
+---
+
+<p class='note'>
+To get AVM fritzhome switch follow the instructions for the general [Fritzhome](/components/fritzhome/).
```
Comment (c727) [inline source/_components/switch.fritzhome.markdown]: Fritzbox

**Comment by c727** [comment]
Also rename the files to the new name, please

**Comment by frenck** [comment]
Merged in origin/next in order to fix the build issues.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
👍 

Can be merged as soon as the parent PR has been merged.

**Review by frenck — APPROVED** [review]

**Pull request merged**